src/ui/domemittermixin.js

```diff
@@ -254,6 +254,15 @@ function getNodeUID( node ) {
 	return node[ 'data-ck-expando' ] || ( node[ 'data-ck-expando' ] = utils.uid() );
 }
 
+// Checks if given node is native DOM Node.
+//
+// @private
+// @param {Node} node
+// @return {Boolean} True when native DOM Node.
+function isDomNode( node ) {
+	return node && node.nodeType && node.nodeName;
```

Comment:
> Yes.

I didn't ask about the theory :P. I asked about the practice. But let's assume that we're safe here.